script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6.txt

```diff
@@ -22,7 +22,7 @@ I can't help but feel embarrassed as she serves me politely with a smile on her
 It's not because of my life at the Arima house and such. I'm just lower middle class at heart.//　有間の家での生活云々ではなく、きっと根が小市民なのだ、俺は。liberally localized last clause: lit. "surely, the root is that I'm lower middle class."
 <Page8>
 "I will go talk to Akiha-sama, please wait until then.# It will probably take about ten minutes, so you may want to look around the mansion until then."
-"Is it fine for me to selfishly walk around on my own?"
+"Is it fine for me to selfishly walk around on my own?" //Is "selfish" really the best word for 勝手? It reads oddly.
```

Comment:
> Is it really fine for me to just wander around as I please?

Reply:
on line 27 Kohaku repeats it though, so maybe a concrete word was chosen to allow for the repetition

Reply:
If you accept change above perhaps change L27 to 
> What are you talking about! This is your home now, how could there be anything wrong with you walking around it!

Reply:
Correct, I was using selfish because otherwise 27 didn't make a lot of sense. This is a good fix